src/pages/language-tour/custom-types.mdx

```diff
@@ -228,7 +228,7 @@ They are most useful for simplifying type signatures.
 type Person = (String, Integer)
 
 fn create_person(name: String, age: Integer) -> Person {
-    #(name, age)
+    (name, age)
```

Comment:
Hey! The typo is actually on the line above, it should be:

```
type Person = #(String, Integer)
```

Reply:
It's a bit weird no ? The primitive tuple is declared without the # ...

Reply:
It is not atm

Reply:
Oopsie. We didn't update the doc regarding tuples.